src/components/ParticipantDeleteErrorModal/ParticipantDeleteErrorModal.i18n/ru.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Cannot delete participant now": "Невозможно удалить участника",
+    "The user has actual goals": "На пользователя назначены цели",
+    "Before delete a participant, you must move it`s goals to other person:": "Перед удалением участника из списка назначте его цели другим пользователям:",
```

Comment:
Before deleting a participant, you must move their goals to another person

Reply:
fixed